handler/setup.go

```diff
@@ -36,8 +37,11 @@ func HandleSetup(engine *engine.Engine) http.HandlerFunc {
 			WriteBadRequest(w, err)
 			return
 		}
-
-		collector := osstats.New(context.Background(), statsInterval)
+		logProcess := false
+		if val, ok := s.Envs[harnessEnableDebugLogs]; ok && val == "true" {
```

Comment:
where is this variable being populated?

Reply:
didn't get, it will come as stage variable so it'll be part of envs

Reply:
I thought this was coming from runner or ci-manager. clear now.